package.json

```diff
@@ -214,10 +215,10 @@
     "terser-webpack-plugin": "2.0.1",
     "url-loader": "1.1.2",
     "url-polyfill": "1.1.3",
-    "webpack": "^4.39.1",
+    "webpack": "4.41.2",
     "webpack-bundle-analyzer": "3.0.3",
     "webpack-cli": "^3.3.8",
-    "webpack-dev-server": "^3.8.1"
+    "webpack-dev-server": "3.8.2"
```

Comment:
hey Vit, I noticed that you update these packages version. what is perfect but why you did it? I mean did you check this with Yashar? is just for avoid any future problem.

Reply:
These changes come from `npm update` because there was some warning telling me to do that. Is it ok @yasharAyari ? 

If not, I can just revert https://github.com/LiskHQ/lisk-hub/pull/2569/commits/212f06fa734c8ff667d4fc7e26217a827c893e4d